cli/bpmetadata/types.go

```diff
@@ -62,9 +62,10 @@ type BlueprintActuationTool struct {
 }
 
 type BlueprintDescription struct {
-	Tagline   string `json:",omitempty" yaml:",omitempty"`
-	Detailed  string `json:",omitempty" yaml:",omitempty"`
-	PreDeploy string `json:"preDeploy,omitempty" yaml:"preDeploy,omitempty"`
+	Tagline      string `json:",omitempty" yaml:",omitempty"`
+	Detailed     string `json:",omitempty" yaml:",omitempty"`
+	PreDeploy    string `json:"preDeploy,omitempty" yaml:"preDeploy,omitempty"`
+	Architecture string `json:"architecture,omitempty" yaml:"architecture,omitempty"`
```

Comment:
Should we make this a []string and require an ordered list in md?

Reply:
Yes, once the implementation is changed, I will change this to an array

Reply:
Ended up changing it with this release.